app/validation/error_messages.py

```diff
@@ -22,4 +22,8 @@
     'MAX_LENGTH_EXCEEDED': 'Your answer is too long, it has to be less than %(max)d characters.',
     'INVALID_DATE': 'Enter a valid date.',
     'INVALID_DATE_RANGE': "Enter a 'period to' date later than the 'period from' date.",
+    'DATE_PERIOD_TOO_SMALL': 'Enter a reporting period greater than %(min)s.',
```

Comment:
I'm sure this conversation has already happened - but why are we hard-coding a specific error message about reporting periods when the validation is generic date range validation?

Reply:
Speaking to Will he just said to use this for now, and then we can change as we need